src/components/ArtistName.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+export const ArtistName = ({ artistList }) => {
+  return (
+    <div className="artist-name-list">
+      {artistList
+        .map((artist) => (
+          <a
+            href={artist.external_urls.spotify}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="artist-name"
+            key={artist.id}
+          >
+            {artist.name}
+          </a>
+        ))
+        .reduce((current, artist, currentIndex) => [
+          current,
+          currentIndex === artistList.length - 1 ? " & " : ", ",
```

Comment:
We noticed that this makes the first artist show the artist name three times. Like Capo1, Capo2, Capo3. Do you know why this happens?//Sofie and Alma